posix.mak

```diff
@@ -506,6 +506,9 @@ $W/dmd-%.html : %.ddoc dcompiler.dd $(DDOC) $(DMD)
 $W/dmd-%.verbatim : %.ddoc dcompiler.dd verbatim.ddoc $(DMD)
 	$(DMD) -c -o- -Df$@ verbatim.ddoc dcompiler.dd $<
 
+$W/install.sh:
+	wget https://raw.githubusercontent.com/dlang/installer/stable/script/install.sh -O web/install.sh
```

Comment:
I think #2033 would be better. The git clone is cheap anyways and more reliable. This is already done for the changelog script anyways.

Reply:
Me too, though obviously we need @MartinNowak to confirm.